src/networking/CMakeLists.txt

```diff
@@ -1,135 +1,122 @@
-# LLNS Copyright Start
-# Copyright (c) 2014-2018, Lawrence Livermore National Security
-# This work was performed under the auspices of the U.S. Department
-# of Energy by Lawrence Livermore National Laboratory in part under
-# Contract W-7405-Eng-48 and in part under Contract DE-AC52-07NA27344.
-# Produced at the Lawrence Livermore National Laboratory.
-# All rights reserved.
-# For details, see the LICENSE file.
-# LLNS Copyright End
-
-set(networking_sources
-	networkingInterface.cpp)
-set(networking_headers
-	networkingInterface.h)
-
-if ((ZMQ_ENABLE) OR (ZMQ_NEEDED))
-set(zmqLibrary_sources
-	zmqLibrary/zmqContextManager.cpp
-	zmqLibrary/zmqHelper.cpp
-	zmqLibrary/zmqReactor.cpp
-	zmqLibrary/zmqProxyHub.cpp
-	zmqLibrary/zmqSocketDescriptor.cpp
-	)
-
-set(zmqLibrary_headers
-	zmqLibrary/zmqContextManager.h
-	zmqLibrary/zmqHelper.h
-	zmqLibrary/zmqReactor.h
-	zmqLibrary/zmqProxyHub.h
-	zmqLibrary/zmqSocketDescriptor.h
-	)
-
-set(zmqInterface_sources
-	zmqCommunicator.cpp
-	zmqInterface.cpp
-	)
-
-set(zmqInterface_headers
-	zmqCommunicator.h
-	zmqInterface.h
-	)
-
-list(APPEND networking_sources ${zmqLibrary_sources})
-list(APPEND networking_headers ${zmqLibrary_headers})
-list(APPEND networking_sources ${zmqInterface_sources})
-list(APPEND networking_headers ${zmqInterface_headers})
+# LLNS Copyright Start Copyright (c) 2014-2018, Lawrence Livermore National
+# Security This work was performed under the auspices of the U.S. Department of
+# Energy by Lawrence Livermore National Laboratory in part under Contract
+# W-7405-Eng-48 and in part under Contract DE-AC52-07NA27344. Produced at the
+# Lawrence Livermore National Laboratory. All rights reserved. For details, see
+# the LICENSE file. LLNS Copyright End
```

Comment:
We may have to reformat the copyright statements to make them work right in the formatter